### ISSUE TITLE
IUT fails to read local path under TEAM Engine

### ISSUE THREAD

**Issue by openfirmware** [opengeospatial/ets-cdb10#26]
Using the absolute file path as the IUT, the TestNG test methods pass when using Eclipse as the environment. But under TEAM Engine, many tests fail as they cannot read the "Metadata" directory. After some debugging, I found that only under TEAM Engine does the IUT have an additional `file:` added to the IUT path. This appears to wrong, as instead of reading an absolute path, a relative path is used and a directory named `file:/` is created in the Tomcat working directory.

I will check to see where `file:` is coming from, and see if using `file://` as the prefix works with Java's file APIs.

**Comment by ghobona** [comment]
`file://` is a protocol used to create a URL to a path on the local filesystem. Officially it's called the [file URI scheme](https://tools.ietf.org/id/draft-kerwin-file-scheme-07.html).

**Comment by openfirmware** [comment]
I added some code to `CommonFixture` to remove `file://` from the IUT if present, and this solves the issue of the test suite in TEAM Engine not reading the IUT properly.

However, this has revealed that the CDB reference files are not being "installed" into the JAR file. This means test methods that check XML reference files for valid ranges (and so on) are skipped. I modified the `pom.xml` file to include the `src/test/resources/CDB` directory for reference, but this has a new problem.

The code that refers to reference files is designed to read the files from disk using path strings, but that does not work when the reference source files are in a JAR file. Instead an `InputStream` is used to load the XML Document, which works, but the Metadata XML classes have mixed usage of how XML Documents are loaded and searched. I don't think the same classes should be used for reading from a CDB under test as for reading from the CDB reference files, because there is so much passing of "path" and filename strings. I plan to create a new set of classes for accessing reference values in tests instead of mixing them with the `org/opengis/cite/cdb10/util/metadataXml` classes.
